pkg/agent/openflow/network_policy.go

```diff
@@ -1763,6 +1768,9 @@ func newFeatureNetworkPolicy(
 		enableDenyTracking:       enableDenyTracking,
 		enableAntreaPolicy:       enableAntreaPolicy,
 		category:                 cookie.NetworkPolicy,
+		connectUplinkToBridge:    connectUplinkToBridge,
+		ctZoneSrcFieldName:       getZoneSrcFieldName(connectUplinkToBridge),
```

Comment:
Why not calculate the result in the caller and pass it when calling function newFeatureNetworkPolicy?

Reply:
This will be only called once when initializing, and this field is only affected by `connectUplinkToBridge`, thus we can pass `connectUplinkToBridge` and calculate ctZone parameters here.